docs/en/observability/monitor-infra/analyze-hosts.asciidoc

```diff
@@ -62,6 +62,15 @@ hosts specified by the host limit).
 and include (or exclude) metrics for those hosts.
 ** In the **Cloud Provider** list, select one or more cloud providers to
 include (or exclude) metrics for the selected cloud providers.
```

Comment:
This update adds a nice explanation to to the Service name filter that explains what it does (thanks, @roshan-elastic)

> This filter is useful for comparing different hosts to determine whether a problem lies
with a service or the host that it is running on.

What can we say about Operating System and Cloud Provider? Is the use case similar, i.e. to determine if the problem lies within the service or the System/Provider?

Reply:
Hey Brandon, that's a good point. I think they're fairly self-explanatory but prob worth adding (maybe even worth adding in the UI as tooltips too actually):

**Operating System**
Filter your hosts by which operating system running on them.

**Cloud Provider**
Filter by the cloud provider your host is running on.